src/plugins/index_pattern_editor/public/components/rollup_beta_warning/rollup_beta_warning.tsx

```diff
@@ -23,17 +23,17 @@ export const RollupBetaWarning = () => (
   <EuiCallOut title={rollupBetaWarningTitle} color="warning" iconType="help">
     <p>
       <FormattedMessage
-        id="indexPatternEditor.rollupIndexPattern.warning.textParagraphOne"
-        defaultMessage="Kibana's support for rollup index patterns is in beta. You might encounter
-issues using these patterns in saved searches, visualizations, and dashboards. They
+        id="indexPatternEditor.rollupDataView.warning.textParagraphOne"
+        defaultMessage="Kibana's support for rollup data view is in beta. You might encounter
+issues using these in saved searches, visualizations, and dashboards. They
```

Comment:
I'm not sure, but something seems odd about "support for rollup data view is in beta."

```suggestion
        id="indexPatternEditor.rollupDataView.warning.textParagraphOne"
        defaultMessage="Kibana's support for data views based on rollups is in beta. You might encounter
issues using these in saved searches, visualizations, and dashboards. They
```